andrea/js/app.js

```diff
@@ -0,0 +1,67 @@
+// buttons //
+var signUp = document.querySelector(".navbar-sign");
+var logIn = document.querySelector(".navbar-login");
+
+var create_btn = document.querySelector(".create-btn");
+
+// form //
+var form = document.querySelector(".register-form");
+
+// passwords //
+var password = document.getElementById("password");
+var confirm_password = document.getElementById("confirm-password");
+
+// show register form //
+function showRegister() {
+    document.querySelector(".title").style.display = "none";
+    document.querySelector(".description").style.display = "none";
```

Comment:
less code is better code, what if you create a container for the title and the description and instead of setting the display to none in 2 elements just do it for a container? This way you only manipulate the style of a single element.